lib/tasks/users.rake

```diff
@@ -156,6 +156,7 @@ namespace :users do
     raise "Couldn't find organisation: '#{args.org}'" unless organisation
 
     users_to_update = User.where(organisation_id: organisation.id, require_2sv: false)
+                          .where("reason_for_2sv_exemption IS NULL")
```

Comment:
This can also be written as `where(reason_for_2sv_exemption: nil)`.  Either work, but the `nil` is more Rails-y.

Reply:
Oh good shout, I'd copied this is not when I was checking for not null, will amend